src/code/z_view.c

```diff
@@ -361,11 +350,8 @@ s32 View_RenderToOrthographicMatrix(View* view) {
 
     gfxCtx = view->gfxCtx;
 
-    {
-        Vp* _vp = (Vp*)gfxCtx->polyOpa.d - 1;
-        vp = _vp;
-        gfxCtx->polyOpa.d = (Gfx*)_vp;
-    }
+    vp = GRAPH_ALLOC(gfxCtx, sizeof(Vp) * 1);
```

Comment:
The multiplication here is unnecessary. Was it needed to match?

Reply:
It wasn't needed to match, but I thought it looked better and kept everything consistent. I think it's easier for someone browsing the code to see it's a single element when the format matches all of the others.

Reply:
I prefer the times 1 is removed, even for Gfx, but it's not a big preference. I prefer consistency. I don't really agree that the type makes that information more helpful.

If you feel strongly the opposite way, let's go that way. My preference is small.